tern/formats/spdx/spdxtagvalue/generator.py

```diff
@@ -235,8 +235,6 @@ def generate(self, image_obj_list):
         # The image's PackageDownloadLocation is from a container registry
         # This includes all the layers but the packages' download location
         # is unknown if the download_url is blank
-        registry_repotag = image_obj.get_download_location() if hasattr(
```

Comment:
@manaswinidas Since we are removing the registry_repotag assignment, could you also remove the comment above it?